KotlinFutures/app/src/main/java/com/example/kotlinfutures/model/RepositoryDefault.kt

```diff
@@ -0,0 +1,38 @@
+package com.example.kotlinfutures.model
+
+import android.util.Log
+import com.example.kotlinfutures.model.network.ApiAccess
+import com.example.kotlinfutures.model.network.Retrofit
+import com.example.kotlinfutures.model.poko.UserResponse
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
+import kotlin.coroutines.suspendCoroutine
+
+class RepositoryDefault : Repository {
+
+    private val baseUrl = "https://randomuser.me/"
```

Comment:
It would be great to have separate file for constants related to all REST requests